### DISCUSSION TITLE
FLOSS TTS engine recommendation ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [pilot51/voicenotify#130] [Q&A] [answered]
Hi,

I believe `voicenotify` come without a TTS engine.

As I don't have one installed. Do you where I could found one FLOSS ?

Thanks.

**Comment by pilot51** [comment, marked as answer]
The only open source engine for Android that I'm aware of is [eSpeak NG](https://github.com/espeak-ng/espeak-ng).

Its quality leaves a lot to be desired compared to most proprietary engines like Google TTS, but if FLOSS is what you're after, eSpeak is what you want.

Don't trust the eSpeak published on Google Play, it uses a different package name than what's on GitHub and has ads and in-app purchases.
Download the APK directly from GitHub [releases](https://github.com/espeak-ng/espeak-ng/releases) or from [F-Droid](https://f-droid.org/packages/com.reecedunn.espeak).

**Reply by BobSquarePants** [reply]
Thank you @pilot51 !

Just a little warning for those how use **Android 14** https://github.com/espeak-ng/espeak-ng/issues/1862
